lib/screens/explore_screens/regions_screens/regions_main/components/region_event_card/interactor/viewmodels/region_event_card_state.dart

```diff
@@ -2,19 +2,20 @@ part of 'region_event_card_bloc.dart';
 
 class RegionEventCardState extends Equatable {
   final List<ProfileModel> profiles;
-  final bool isEventPassEndDate;
+  final RegionEventModel event;
 
-  const RegionEventCardState({required this.profiles, required this.isEventPassEndDate});
+  const RegionEventCardState({required this.profiles, required this.event});
 
   @override
-  List<Object> get props => [profiles, isEventPassEndDate];
+  List<Object> get props => [profiles, event];
```

Comment:
If the event in state will be nerver modified is not necessary put it in the props. Its just an note, it does not hurts have it too.